.github/workflows/dockerhub-push.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Docker image and push to DockerHub
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/gotestwaf:latest
```

Comment:
I'm not sure that `${{ secrets.DOCKERHUB_USERNAME }}` should be placed here, probably just `wallarm`

Reply:
Yes, in case of `vault-action` we need to change it to `wallarm`.

Reply:
Fixed - https://github.com/wallarm/gotestwaf/pull/185/commits/3d9a4babbaa744126d14d9e9af648b513afa84dc